crowbar_framework/lib/crowbar/logger/formatter.rb

```diff
@@ -18,20 +18,55 @@ module Crowbar
   module Logger
     class Formatter < ::Logger::Formatter
       include ActiveSupport::TaggedLogging::Formatter
+      FORMAT = "%s, [%s#%d] %5s -- %s\n"
+      THREAD_FORMAT = "%s, [%s#%d:%s] %5s -- %s\n"
 
-      def call(severity, timestamp, progname, msg)
-        final_message = if msg.is_a? String
+      def call(severity, time, progname, msg)
+        if (ENV["CROWBAR_THREADS"].to_i || 10) > 1
```

Comment:
If `CROWBAR_THREADS` is not set (empty string or `nil`), or is set to 0, is expected to have 10 threads? Also if this is the puma default, I also expect to remove this magic number from the check.